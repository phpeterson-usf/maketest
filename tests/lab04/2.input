-e "0b1011101011010001 - 0b10101" -b 2
