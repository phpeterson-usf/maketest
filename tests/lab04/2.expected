0b00000000000000001011101010111100
